### ISSUE TITLE
pvalues only correct for one level

### ISSUE THREAD

**Issue by lboxell** [kolesarm/Robust-Small-Sample-Standard-Errors#2]
Hi @kolesarm,

We were using this package for one of our projects and we noticed that the p-values that get produced by `dfadjustSE` don't appear to be correct. I believe the formula [here](https://github.com/kolesarm/Robust-Small-Sample-Standard-Errors/blob/master/R/ols.R#L177) should be 
`2*stats::pt(-abs(x$coefficients[, "Estimate"] / x$coefficients[, "HC2 se"]), x$coefficients[, "df"])`. I believe the current p-value that gets produced is only accurate if the true p-value happens to be 0.05. 

Maybe we're mistaken or this p-value was serving a separate purpose.

Simple example:
```
library(dfadjust)
data("mtcars")
x <- dfadjustSE(lm(mpg ~ hp, data = mtcars))

2 * stats::pt(-abs(x$coefficients[, "Estimate"] / x$coefficients[, "HC2 se"]), x$coefficients[, "df"])

2 * stats::pnorm(-abs(x$coefficients[, "Estimate"] / x$coefficients[, "Adj. se"]))
```

**Comment by kolesarm** [comment]
It's actually the other way around: the p-values are correct, but you have to be careful when constructing confidence intervals with level that is different from 95%. This is because we use a degrees of freedom correction to the critical value (as the package name suggests): we use a t-distribution with a data-dependent degrees of freedom. 

The standard error that is reported is an "adjusted standard error", as explained in the manual:  it is the HC2 standard error multiplied by `qt(0.975, df=dof)/qnorm(0.975)` so that one can construct 95\% confidence intervals by adding and subtracting 1.96 times the adjusted standard error. We do this to make the standard errors comparable across different inference methods.

**Comment by lboxell** [comment]
Hi @kolesarm,

Thanks for the response. @jmshapir and I wanted to follow up on this to clarify the question. 

Here is a pdf outlining our understanding and our question: [ikpvalues.pdf](https://github.com/kolesarm/Robust-Small-Sample-Standard-Errors/files/6951582/ikpvalues.pdf)

And here is a screenshot of it:
<img width="726" alt="Screen Shot 2021-08-08 at 7 02 47 PM" src="https://user-images.githubusercontent.com/13038185/128648228-a99223a7-ad25-4e1e-a2a7-c14835c992d1.png">

Thanks in advance!

**Comment by kolesarm** [comment]
I am sorry, I misunderstood, I thought you were referring to the `dfadjustSE` function, rather than the `print` method. You are right, there was a bug in the print method, it should now be fixed in the latest commit [#36f6722](https://github.com/kolesarm/Robust-Small-Sample-Standard-Errors/commit/36f6722c9de1e3a4f68aca144dca852a56acc80d), see [lines 176-179](https://github.com/kolesarm/Robust-Small-Sample-Standard-Errors/blob/master/R/ols.R#L176-L179). 

Now  
```{R}
print(dfadjustSE(lm(mpg ~ hp, data = mtcars)))
``` 
should return

```
Coefficients:
              Estimate    HC1 se    HC2 se   Adj. se       df     p-value
(Intercept) 30.0988605 2.0766149 2.1930119 2.3888632 14.71874 8.64162e-10
hp          -0.0682283 0.0135604 0.0147147 0.0168347  9.55342 1.04575e-03
```

**Comment by lboxell** [comment]
Thanks @kolesarm! And, no worries, I could have been clearer initially.
